### PR TITLE
Improve Coral Edge TPU guide

### PR DESCRIPTION
**Pull request by Skillnoob**
This pr contains some small improvements suggested to me by @Burhan-Q .
This should also fix the author being wrong in the guide.

I have read the CLA Document and I hereby sign the CLA

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary
Guide update for using Coral Edge TPU with Ultralytics on Raspberry Pi. 🚀

### 📊 Key Changes
- Updated the description to reflect the guide's purpose more clearly.
- Renamed sections for better clarity and context.
- Provided a new guide to replace the outdated Coral guide, ensuring compatibility with the latest TensorFlow Lite runtime versions.
- Added an updated link for downloading the latest version of the Coral Edge TPU runtime.
- Simplified the installation command for the Edge TPU runtime.
- Included detailed commands for both model export and prediction using the Edge TPU.

### 🎯 Purpose & Impact
- **Enhanced Clarity:** Provides clearer instructions, helping users successfully set up Ultralytics with a Coral Edge TPU on a Raspberry Pi.
- **Performance Improvement:** Aims at improving inference performance on resource-constrained devices like Raspberry Pi.
- **Compatibility Maintenance:** Updates ensure smooth operation with the latest software, given Google's apparent abandonment of the Coral project.
- **User Empowerment:** By providing an alternative guide, users can continue leveraging the Coral Edge TPU for machine learning applications despite the lack of official updates.

### CODE REVIEW

**Comment by Burhan-Q** [comment]
All looks good to me ☄️ thanks @Skillnoob ! CI error due to Google Drive quota limit and mentioned in #8161 for discussion about fixing or changing

**Review by Burhan-Q — COMMENTED** [review]
Pedantic comment 😆, no change _required_

File: docs/en/guides/coral-edge-tpu-on-raspberry-pi.md [hunk 1]
```diff
@@ -69,9 +70,9 @@ After installing the runtime, you need to plug in your Coral Edge TPU into a USB
     sudo apt remove libedgetpu1-max 
     ```
 
-## Exporting your model to a Edge TPU compatible model
+## Export your model to a Edge TPU compatible model
 
-To use the Edge TPU, you need to convert your model into a compatible format. It is recommended that you run export on Google Colab, x86_64 Linux machine, using the official [Ultralytics Docker container](docker-quickstart.md), or using [Ultralytics HUB](../hub/quickstart.md), since the Edge TPU compiler is not available on ARM. See the [Export Mode](../modes/export.md) for the available arguments.
+To use the Edge TPU, you need to convert your model into a compatible format. It is recommended that you run export on Google Colab, x86_64 Linux machine, using the official [Ultralytics Docker container](docker-quickstart.md) or using [Ultralytics HUB](../hub/quickstart.md), since the Edge TPU compiler is not available on ARM. See the [Export Mode](../modes/export.md) for the available arguments.
```
[the remark this answers is not in the document]
Comment (Skillnoob) [reply docs/en/guides/coral-edge-tpu-on-raspberry-pi.md:75]: Checked over it with a grammar checker and you're correct. Will push a fix in a second

**Comment by glenn-jocher** [comment]
@Skillnoob @Burhan-Q awesome, thanks for the updates! This should reflect the correct author now on Docs deploy in a few min :)

**Pull request merged**